apps/web/lib/component-suggestions.ts

```diff
@@ -0,0 +1,157 @@
+/**
+ * Component suggestions for EditableHint components
+ * This file contains suggestion arrays for different components across the app
+ */
+
+export interface ComponentSuggestion {
+  id: string;
+  title: string;
+  detailedSuggestion: string;
+  messageId: string;
+}
+
+/**
+ * Suggestions for the Custom Instructions Editor component
+ */
+export const customInstructionsEditorSuggestions: ComponentSuggestion[] = [
```

Comment:
In general it's an anti-pattern to gather stuff for the different components, which mostly don't know about each other, and put them all in one file. Instead, these instructions should be distributed out to each component's files. 

With this pattern, trying to render one component effectively brings in all the suggestions for all the components, so they're no longer independent...